Rakefile

```
@@ -19,4 +19,36 @@ desc 'Generate code coverage'
 RSpec::Core::RakeTask.new(:coverage) do |t|
   t.rcov = true
   t.rcov_opts = ['--exclude', 'spec']
-end
\ No newline at end of file
+end
+
+desc 'Executes reference tests (agent only) intended for use in CI'
+task :reference_tests do
+    command =<<-EOS
+bundle exec beaker                          \
+    --debug                                 \
+    --preserve-hosts never                  \
+    --config tests/configs/$PLATFORM        \
+    --keyfile ~/.ssh/id_rsa-acceptance      \
+    --load-path tests/lib                   \
+    --type aio                              \
+    --pre-suite tests/reference/pre-suite   \
+    --tests tests/reference/tests
+    EOS
+    sh command
+end
+
+desc 'Executes accetpance tests (master and agent) indened for use in CI'
+task :acceptance_tests do
+    command =<<-EOS
+bundle exec beaker
+    --debug                                 \
+    --preserve-hosts never                  \
+    --config tests/configs/$PLATFORM        \
```

Let's make sure the task will fail with a descriptive message if the $PLATFORM envar is not set.